dirsrvtests/tests/suites/password/pwdPolicy_logging_test.py

```diff
@@ -0,0 +1,152 @@
+import ldap
+import logging
+import pytest
+import os
+import time
+from lib389._constants import DEFAULT_SUFFIX, PASSWORD
+from lib389.topologies import topology_st as topo
+from lib389.pwpolicy import PwPolicyManager
+from lib389.idm.organizationalunit import OrganizationalUnits
+from lib389.idm.user import UserAccount, UserAccounts, TEST_USER_PROPERTIES
+from lib389.idm.domain import Domain
+
+log = logging.getLogger(__name__)
+
+LOCAL_RDN = 'ou=People'
+GLOBAL_RDN = 'ou=Global'
+LOCAL_BIND_DN ="uid=local_user,ou=people," + DEFAULT_SUFFIX
+GLOBAL_BIND_DN ="uid=global_user,ou=global," + DEFAULT_SUFFIX
+
+
+def create_entries(inst):
+    # Create local user
+    users = UserAccounts(inst, DEFAULT_SUFFIX, rdn=LOCAL_RDN)
+    user_props = TEST_USER_PROPERTIES.copy()
+    user_props.update({'uid': 'local_user', 'cn': 'local_user', 'userpassword': PASSWORD})
+    users.create(properties=user_props)
+
+    # Create new OU
+    ou_global = OrganizationalUnits(inst, DEFAULT_SUFFIX).create(properties={'ou': 'Global'})
+
+    # Create global user
+    users = UserAccounts(inst, DEFAULT_SUFFIX, rdn=GLOBAL_RDN)
+    user_props = TEST_USER_PROPERTIES.copy()
+    user_props.update({'uid': 'global_user', 'cn': 'global_user', 'userpassword': PASSWORD})
+    users.create(properties=user_props)
+
+    # Add aci
+    aci = '(targetattr="*")(version 3.0; acl "pwp test"; allow (all) userdn="ldap:///self";)'
+    suffix = Domain(inst, DEFAULT_SUFFIX)
+    suffix.add('aci', aci)
+
+
+def create_policies(inst):
+    # Configure subtree policy
+    pwp = PwPolicyManager(inst)
+    subtree_policy_props = {
+        'passwordCheckSyntax': 'on',
+        'passwordMinLength': '6',
+        'passwordChange': 'on',
+        'passwordLockout': 'on',
+        'passwordMaxFailure': '2',
+    }
+    pwp.create_subtree_policy(f'{LOCAL_RDN},{DEFAULT_SUFFIX}', subtree_policy_props)
+
+    # Configure global policy
+    inst.config.replace('nsslapd-pwpolicy-local', 'on')
+    inst.config.replace('passwordCheckSyntax', 'on')
+    inst.config.replace('passwordMinLength', '8')
+    inst.config.replace('passwordChange', 'on')
+    inst.config.replace('passwordLockout', 'on')
+    inst.config.replace('passwordMaxFailure', '5')
+    time.sleep(1)
+
+
+def test_debug_logging(topo):
+    """Enable password policy logging
+
+    :id: cc152c65-94e0-4716-a77c-abdd2deec00d
+    :setup: Standalone Instance
+    :steps:
+        1. Set password policy logging level
+        2. Add database entries
+        3. Configure local and global policies
+        4. Test syntax checking on local policy
+        5  Test syntax checking on global policy
```

Comment:
Oh, sorry, I think, the issue is in this line.
It should be `5. Test syntax checking on global policy` - so you need to add a dot after `5`, otherwise, the Validate test fails. Ugh...